json/json_parse_stream.ts

```diff
@@ -17,48 +17,44 @@ function isBrankString(str: string) {
  * {@link https://www.rfc-editor.org/rfc/rfc7464.html | JSON Text Sequences}.
  * Chunks consisting of spaces, tab characters, or newline characters will be ignored.
  *
- * @example
- * parse JSON lines or NDJSON
- * ```ts
- * import { TextLineStream } from "@std/streams/text-line-stream";
- * import { JsonParseStream } from "@std/json/json-parse-stream";
- *
- * const url = "@std/json/testdata/test.jsonl";
- * const { body } = await fetch(url);
- *
- * const readable = body!
- *   .pipeThrough(new TextDecoderStream())  // convert Uint8Array to string
- *   .pipeThrough(new TextLineStream()) // transform into a stream where each chunk is divided by a newline
- *   .pipeThrough(new JsonParseStream()); // parse each chunk as JSON
```

Comment:
Can we keep one of this type of example with `no-eval`? This seems typical real world usage of this class to me. It helps the users to know how to read JSON Lines response from servers